src/internet_message.c

```diff
@@ -163,22 +160,26 @@ static int im_header_add_fold_ctx(struct im_header_context *ctx)
  */
 int im_header_feed(struct im_header_context *ctx, char c)
 {
+	jsval header;
```

Comment:
The value of header seems to be uninitialized and lost between successive calls to im_header_feed(). Maybe the value doesn't get overwritten on the stack and (since the address/layout of the stack frame is always the same) it's still there at the next call. I guess 'header' should live inside ctx instead.